test/cypress/integration/group-chat-direct-message.spec.js

```diff
@@ -233,32 +215,37 @@ describe('Create/Join direct messages and orders of direct message channels', ()
     })
     cy.getByDT('addPeople').click()
     cy.getByDT('unjoinedChannelMembersList').within(() => {
-      cy.getByDT('addToChannel-' + user4).click()
+      cy.getByDT('addToChannel-' + user1).click()
     })
 
-    cy.getByDT('channelName').should('contain', `${user3}, ${user4}`)
+    cy.getByDT('channelName').should('contain', `${user3}, ${user1}`)
+    cy.giWaitUntilMessagesLoaded(false)
     cy.url().then(url => {
       cy.get('@groupMessageLink').should('eq', url) // NOTE: this checks the possibility to create gm for same users
     })
+    cy.getByDT('conversationWrapper').within(() => {
+      cy.get('.c-message:last-child').should('contain', message)
+    })
   })
 
-  it('user3 adds user5 to the channel again', () => {
+  it('user3 adds user2 to the DM, so all group members are in the same DM', () => {
     switchUser(user3)
     cy.giRedirectToGroupChat()
 
-    openDMByMembers([user1, user4])
+    openDMByMembers([user4, user1])
 
     cy.getByDT('channelName').within(() => {
       cy.getByDT('menuTrigger').click()
     })
     cy.getByDT('addPeople').click()
     cy.getByDT('unjoinedChannelMembersList').within(() => {
-      cy.getByDT('addToChannel-' + user5).click()
+      cy.getByDT('addToChannel-' + user2).click()
     })
-    cy.getByDT('closeModal').click()
 
-    cy.getByDT('channelName').should('contain', `${user1}, ${user4}, ${user5}`)
-    cy.getByDT('conversationWrapper').find('.c-message').should('have.length', 4) // 3 means user1, user3, user4, user5
+    cy.getByDT('channelName').should('contain', `${user4}, ${user1}, ${user2}`)
+    cy.giWaitUntilMessagesLoaded(false)
+    // NOTE: no notification messages in DM
+    cy.getByDT('conversationWrapper').find('.c-message').should('have.length', 0)
```

Comment:
Actually, in this test scenario (group-chat-direct-message), it has the similar scenario as the previous version.
Changes are for
- Optimizing the workflow to save time
- Removing `user5` because I found that we don't need 5 members for the test. The more users, the more time takes.